network/src/commonMain/kotlin/com/wire/kalium/network/api/base/authenticated/notification/conversation/MessageEventData.kt

```diff
@@ -28,3 +29,47 @@ data class MessageEventData(
     @SerialName("recipient") val recipient: String,
     @SerialName("data") val encryptedExternalData: String? = null
 )
+
+@Serializable
+data class UnEncryptedMessageEventData(
```

Comment:
if those events are related to the backup then it should be isolated in its own classes and not in network, because we will never receive such event from DB

Reply:
Removed